changelogs/current.yaml

```diff
@@ -90,6 +90,11 @@ minor_behavior_changes:
     The redis network filter :ref:`connection_rate_limit_per_sec
     <envoy_v3_api_field_extensions.filters.network.redis_proxy.v3.RedisProxy.ConnectionRateLimit.connection_rate_limit_per_sec>`
     must be greater than 0. A config that sets this value to 0 will be rejected.
+- area: upstream
+  change: |
+    Deprecate code path of legacy upstream load balancer. Ideally, this is implementation detail changes and should not
+    affect users. However, if there is any user who encounters issues, this behavior can be reverted by setting runtime flag
+    ``envoy_reloadable_features_convert_legacy_lb_config`` to false.
```

Comment:
When we trigger an unexpected LB bugs or behavior change. This flag allow users to work around the LB change temporarily.
Because we have no 100% confidence to the new code path and the LB is very very very important.